client/src/app/core/plugins/plugin.service.ts

```diff
@@ -257,6 +258,11 @@ export class PluginService implements ClientHook {
         return environment.apiUrl + `${pathPrefix}/${plugin.name}/${plugin.version}/static`
       },
 
+      getBaseClientScriptsRoute: () => {
```

Comment:
Why do you need the client scripts path since peertube auto loads your scripts?

Reply:
Well, it was for a web-assembly file, but it can be put in the static assets and lazy-load path scripts are defined in Webpack config via the `publicPath`.

I delete it so.